scripts/platoon-launcher

```diff
@@ -89,7 +89,7 @@ multi-node.'''))
 if __name__ == '__main__':
     args = parse_arguments()
 
-    logs_folder = os.path.join("PLATOON_LOGS", args.experiment_name, time.strftime("%Y-%m-%d_%H-%M-%S"))
+    logs_folder = os.path.join("PLATOON_LOGS", args.experiment_name)
```

Comment:
Could you please revert this? I know it is most helpful when testing the code, but we would not like this if we were running training experiments as we may want to keep logs from previous ones. Or does this provoke an issue with `--output-filename` option in `mpirun`?